metis/lib/path.rb

```diff
@@ -24,8 +24,14 @@ def self.filepath_match
       !x
       end
 
+      def self.remove_double_slashes(path)
+        return ::File.join(path.split("/")) unless path.start_with?("metis://")
+
+        ::File.join("metis://", path.split("metis://").last.split("/"))
+      end
+
       def self.path_from_parts(project_name, bucket_name, file_path)
-        "metis://#{project_name}/#{bucket_name}/#{file_path}"
+        ::File.join("metis://", project_name, bucket_name, file_path.split("/"))
```

Comment:
Won't this cause `metis:///`?

Reply:
```
irb(main):001:0> ::File.join("metis://", "example_project", "bucket", "path/to/file.txt".split("/"))
=> "metis://example_project/bucket/path/to/file.txt"
```

Reply:
:smile: Somehow `::File` figures it out.